draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -82,93 +82,160 @@ Copyright Notice
-   specific constructs.
+   request or negotiate with a network slicing provider to create an
+   instance.
+   The customer can incrementally update its requirements on individual
+   topology elements in the slice instance, e.g., adding or removing a
+   node or link, updating desired bandwidth of a link, and retrieve the
+   operational states of these elements.  With the help of other
+   mechanisms and data models defined in IETF, the telemetry information
+   can be published to the customer, too.
+
+
+
+
+
+
+
+
+
+Liu, et al.             Expires 31 December 2023                [Page 3]
+
+Internet-Draft      Network Slice Topology Data Model          June 2023
+
+
+   The YANG model defines constructs that are technology-agnostic to
+   network slicing built on network layers with different technologies
+   such as IP/MPLS, MPLS-TP, OTN and WDM optical.  Therefore, this model
+   can serve as a common and base model on which technology-specific
+   models for network slicing, such as
+   [I-D.ietf-ccamp-yang-otn-slicing], may be built.
 
    As described in Section 3 of
    [I-D.contreras-teas-slice-controller-models], the data model defined
-   in this document complements the data model defined in
-   [I-D.ietf-teas-ietf-network-slice-nbi-yang].  In addition to the
-   provider's view, the data model defined in this document models the
-   Type 2 service defined in [RFC8453].
+   in this document is optional for network slicing and complements the
+   data model defined in [I-D.ietf-teas-ietf-network-slice-nbi-yang].
+   In addition to the provider's view, the data model defined in this
+   document models the Type 2 service defined in [RFC8453].
 
    The YANG data model in this document conforms to the Network
    Management Datastore Architecture (NMDA) [RFC8342].
 
-1.1.  Tree Diagram
+1.1.  Terminologies and Notations
 
-   Tree diagrams used in this document follow the notation defined in
-   [RFC8340].
+   The following terminologies for describing network slices are defined
+   in [I-D.ietf-teas-ietf-network-slices] and are not redefined herein.
 
-1.2.  Prefixes in Data Node Names
+   *  Network Slice (NS)
 
-   In this document, names of data nodes and other data model objects
-   are prefixed using the standard prefix associated with the
-   corresponding YANG imported modules, as shown in Table 1.
+   *  Network Slice Customer
+
+   *  Network Slice Service Provider
+
+   *  Network Slice Controller (NSC)
 
+   *  Network Resource Partition (NRP)
 
+   The following terms are defined and used in this document.
 
+   *  Customized Topology: A topology defined by the customer and served
+      as an input to the network slice service provider, i.e. to the
+      Network Slice Controller (NSC).
 
+   *  Abstract Topology: A topology exposed to the customer by the
+      network slice service provider prior to the creation of network
+      slices.  The provider uses an abstract topology expose useful
```

Comment:
The provider uses an abstract topology **to** expose useful information.
Maybe we can say that "based on these useful information that can be optionally provided, the customer can have buil up his own customized topology."
This is what we have talked today.

Reply:
here the proposed text: 
~~~
A topology exposed to the customer by the network slice service provider prior to
the creation of network slices. The provider uses an abstract topology to expose
useful information, such as available resources to the customer, which can facilitate
the build-up of customized topologies by the customer.
~~~